packages/mui-material/src/ListItem/ListItem.js

```diff
@@ -94,23 +99,23 @@ export const ListItemRoot = styled('div', {
     }),
   }),
   ...(!!ownerState.secondaryAction && {
-    [`& > .${listItemButtonClasses.root}`]: {
+    [`& > .${ownerState.generateListItemButtonClassName('root')}`]: {
       paddingRight: 48,
     },
   }),
-  [`&.${listItemClasses.focusVisible}`]: {
+  [`&.${muiStateClasses.focusVisible}`]: {
```

Comment:
Isn't it worth also getting these global class names through the generator? It would open the possibility to replace the Mui- prefix with something custom, right?